site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -1190,6 +1190,11 @@ private function updateGradeable(Gradeable $gradeable, $details) {
                 $this->core->getQueries()->convertInquiryComponentId($gradeable);
             }
 
+            if ($prop === 'grade_inquiry_per_component_allowed' && $post_val === true && !$gradeable->isGradeInquiryPerComponentAllowed()) {
+                $this->core->getQueries()->revertInquiryComponentId($gradeable);
+                echo "this";
```

Comment:
Code looks clean just remove this debug statement here.

Reply:
Thanks for pointing out, removed :)